config/environments/development.rb

```diff
@@ -11,8 +11,11 @@
   # Do not eager load code on boot.
   config.eager_load = false
 
+  # Raise exceptions instead of rendering exception templates.
+  config.action_dispatch.show_exceptions = true
+
   # Show full error reports.
-  config.consider_all_requests_local = true
+  config.consider_all_requests_local = false
```

Comment:
The application will render the public error pages instead of detailed debug information.

Reply:
It has pros and cons:
It makes it easier to manually test the behaviour as production.
On the other side, for certain debugging, we should set it to false.